### ISSUE TITLE
Problem running the example case in SearchGraph

### ISSUE THREAD

**Issue by Kausal-Lei** [ScrapeGraphAI/Scrapegraph-ai#462]
When I try the example for SearchGraph in https://scrapegraph-doc.onrender.com/docs/Graphs/serch_graph, code as follows:
``` python
from scrapegraphai.graphs import SearchGraph

# Define the prompt and configuration
prompt = "What is Chioggia famous for?"
config = {
    "llm": {
        "api_key": 'sk-xxxx',
        "model": "gpt-3.5-turbo",
    }
}

# Create the search graph
search_graph = SearchGraph(prompt, config)

# Run the search graph
result = search_graph.run()

print(result)
```
I got error as follows:
```
Traceback (most recent call last):
  File "C:\Users\29092\PycharmProjects\crawl\test.py", line 16, in <module>
    result = search_graph.run()
  File "D:\Scrapegraph-ai-1.9.1\scrapegraphai\graphs\search_graph.py", line 126, in run
    self.final_state, self.execution_info = self.graph.execute(inputs)
  File "D:\Scrapegraph-ai-1.9.1\scrapegraphai\graphs\base_graph.py", line 228, in execute
    return self._execute_standard(initial_state)
  File "D:\Scrapegraph-ai-1.9.1\scrapegraphai\graphs\base_graph.py", line 157, in _execute_standard
    raise e
  File "D:\Scrapegraph-ai-1.9.1\scrapegraphai\graphs\base_graph.py", line 144, in _execute_standard
    result = current_node.execute(state)
  File "D:\Scrapegraph-ai-1.9.1\scrapegraphai\nodes\graph_iterator_node.py", line 74, in execute
    state = asyncio.run(self._async_execute(state, batchsize))
  File "D:\ProgramData\Miniconda3\envs\crawl\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "D:\ProgramData\Miniconda3\envs\crawl\lib\asyncio\base_events.py", line 649, in run_until_complete
    return future.result()
  File "D:\Scrapegraph-ai-1.9.1\scrapegraphai\nodes\graph_iterator_node.py", line 133, in _async_execute
    answers = await tqdm.gather(
  File "D:\ProgramData\Miniconda3\envs\crawl\lib\site-packages\tqdm\asyncio.py", line 79, in gather
    res = [await f for f in cls.as_completed(ifs, loop=loop, timeout=timeout,
  File "D:\ProgramData\Miniconda3\envs\crawl\lib\site-packages\tqdm\asyncio.py", line 79, in <listcomp>
    res = [await f for f in cls.as_completed(ifs, loop=loop, timeout=timeout,
  File "D:\ProgramData\Miniconda3\envs\crawl\lib\asyncio\tasks.py", line 571, in _wait_for_one
    return f.result()  # May raise f.exception().
  File "D:\ProgramData\Miniconda3\envs\crawl\lib\site-packages\tqdm\asyncio.py", line 76, in wrap_awaitable
    return i, await f
  File "D:\Scrapegraph-ai-1.9.1\scrapegraphai\nodes\graph_iterator_node.py", line 122, in _async_run
    return await asyncio.to_thread(graph.run)
  File "D:\ProgramData\Miniconda3\envs\crawl\lib\asyncio\threads.py", line 25, in to_thread
    return await loop.run_in_executor(None, func_call)
  File "D:\ProgramData\Miniconda3\envs\crawl\lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "D:\Scrapegraph-ai-1.9.1\scrapegraphai\graphs\smart_scraper_graph.py", line 124, in run
    self.final_state, self.execution_info = self.graph.execute(inputs)
  File "D:\Scrapegraph-ai-1.9.1\scrapegraphai\graphs\base_graph.py", line 228, in execute
    return self._execute_standard(initial_state)
  File "D:\Scrapegraph-ai-1.9.1\scrapegraphai\graphs\base_graph.py", line 157, in _execute_standard
    raise e
  File "D:\Scrapegraph-ai-1.9.1\scrapegraphai\graphs\base_graph.py", line 144, in _execute_standard
    result = current_node.execute(state)
  File "D:\Scrapegraph-ai-1.9.1\scrapegraphai\nodes\fetch_node.py", line 214, in execute
    Document(page_content=parsed_content, metadata={"source": "html file"})
  File "D:\ProgramData\Miniconda3\envs\crawl\lib\site-packages\langchain_core\documents\base.py", line 22, in __init__
    super().__init__(page_content=page_content, **kwargs)
  File "D:\ProgramData\Miniconda3\envs\crawl\lib\site-packages\pydantic\v1\main.py", line 341, in __init__
    raise validation_error
pydantic.v1.error_wrappers.ValidationError: 1 validation error for Document
page_content
  none is not an allowed value (type=type_error.none.not_allowed)

Process finished with exit code 1

```

My python version is 3.10.8, and the version of Scrapegraph-ai is 1.9.1 downloaded from github.

**Comment by VinciGit00** [comment]
have u tried to change llm mode?

**Comment by f-aguzzi** [comment]
I have the feeling that this might be a hidden dependency clash. Errors of all kinds always pop up when trying to use the GitHub version with Conda (I see you have Miniconda installed). Fortunately, the build system used in the making of this project (Rye) provides its own isolated environment. To use it:
1. install [rye](https://rye.astral.sh/) on your machine
2. run `rye sync` from the terminal, in the base folder of your local copy of ScrapeGraph repository
3. a virtual environment should be activated automatically. If it still doesn't work, use `rye run python` instead of calling `python` directly in the terminal. In this case, it would be something like `rye run python test.py`.

In short:
- `conda` (and `venv`) work with the PyPI version of ScrapeGraph (the one you install with pip)
- they don't work well with the source version you can download from GitHub

**Comment by Kausal-Lei** [comment]
I don't try another mode, I found that this might be caused by a url that is only an image, so it crashed. I found it's crashed when got an image.

**Comment by VinciGit00** [comment]
ok
